app/models/cat.rb

```diff
@@ -1,2 +1,5 @@
 class Cat < ApplicationRecord
+  validates :name, :age , :enjoys ,:image, presence: true
+  validates :enjoys ,length: {minimum:10}
```

Comment:
Make sure you don't have extra spaces after your commas.

Reply:
Yes ma'am